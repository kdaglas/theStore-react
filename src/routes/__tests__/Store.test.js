import React from 'react';
import { shallow } from 'enzyme';
import store from '../../Store';


describe('The store', () => {
  it('should reach the store without crushing', () => {
    const component = shallow(<store />);
    expect(component).toMatchSnapshot();
  });
});