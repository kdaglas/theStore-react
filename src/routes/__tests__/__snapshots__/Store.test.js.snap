// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The store should reach the store without crushing 1`] = `ShallowWrapper {}`;
